html/cross-origin-opener-policy/coop-sandbox.https.html

```diff
@@ -29,6 +29,24 @@
   }, `<iframe sandbox="${sandboxValue}"> ${document.title}`);
 });
 
+// Verify the contents of the sandboxing flag set that is used in the
+// navigation check:
+//
+// 1. that it is not the "sandboxing flags" of the source browsing context
+//    (i.e. the iframe)
+// 2. that it was not created by copying the sandboxing flag set
```

Comment:
Basically, we want to verify that the popup doesn't have sandboxing flags set, right? I think it would be clearer to state that.

Reply:
You got it!